xml/en/docs/http/ngx_http_slice_module.xml

```diff
@@ -29,6 +29,23 @@ configuration parameter.
 </section>
 
 
+<section id="issues" name="Known Issues">
+
+<para>
+A slice cannot be updated with background subrequests enabled,
```

Comment:
This line looks too "cache background update"-centric, it should be more generic wrt other modules.